supporter-product-data/cloudformation/cfn.yaml

```diff
@@ -0,0 +1,383 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Product data information for supporters."
+
+Parameters:
+  Stage:
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - PROD
+      - UAT
+      - CODE
+    Default: CODE
+
+Conditions:
+  CreateProdResources: !Equals [!Ref "Stage", "PROD"]
+
+Mappings:
+  StageVariables:
+    CODE:
+      S3Bucket: supporter-product-data-export-code
```

Comment:
kate mentioned today that S3 bucket names shouldn't really be in public repos, although I'm not sure why exactly